From 54be3719e26d21758a4e2a2199f42124419c91f6 Mon Sep 17 00:00:00 2001
From: barracuda156 <vital.had@gmail.com>
Date: Wed, 17 May 2023 16:16:44 +0800
Subject: [PATCH 1/4] Revert "Update methods to
 `dcheck{ToSend,Received}OrEmpty()`"

This reverts commit df79893d4b66e97e9778df8aeea4f712e09a780f.
---
 thrift/lib/cpp2/transport/rocket/PayloadUtils.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git thrift/lib/cpp2/transport/rocket/PayloadUtils.h thrift/lib/cpp2/transport/rocket/PayloadUtils.h
index 9cc7c21cd0..6751a2b351 100644
--- thrift/lib/cpp2/transport/rocket/PayloadUtils.h
+++ thrift/lib/cpp2/transport/rocket/PayloadUtils.h
@@ -187,7 +187,7 @@ rocket::Payload packWithFds(
   auto ret = apache::thrift::rocket::detail::makePayload(
       *metadata, std::move(serializedPayload));
   if (numFds) {
-    ret.fds = std::move(fds.dcheckToSendOrEmpty());
+    ret.fds = std::move(fds.dcheckToSend());
   }
   return ret;
 }
-- 
2.40.1

